ext/mochilo/mochilo_pack.c

```diff
@@ -73,6 +73,56 @@ void mochilo_pack_bignum(mochilo_buf *buf, VALUE rb_bignum)
 	}
 }
 
+#ifdef HAVE_RUBY_ENCODING_H
```

Comment:
This could be saved for another pull request for sure but - I'm pretty sure we dropped support for anything older than Ruby 1.9 so we probably don't need these checks anymore.